client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -2350,6 +2350,34 @@ Undocumented.prototype.transferStatus = function( siteId, transferId ) {
 };
 
 /**
+ * Update the poster for a video.
+ *
+ * @param {string} videoId ID of the video
+ * @param {object} data The POST data
+ * @param {Function} fn Function to invoke when request is complete
+ * @returns {Promise} A promise that resolves when the request is complete
+ */
+Undocumented.prototype.updateVideoPoster = function( videoId, data, fn ) {
```

Comment:
@gwwar This is a WPCOM only endpoint, as VideoPress is a WPCOM feature. Jetpack uploads its videos to WPCOM for all VideoPress requests, not the other way around.